tools/tools.go

```diff
@@ -17,14 +17,14 @@ package tools
 //go:generate go install github.com/evanphx/grpc-gateway/protoc-gen-swagger
 
 //go:generate go install github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-grpc-gateway
-//go:generate go install "-ldflags=-s -w -X github.com/vektra/mockery/mockery.SemVer=2.12.1" github.com/vektra/mockery/cmd/mockery
+//go:generate go install "-ldflags=-s -w -X github.com/vektra/mockery/cmd/mockery.SemVer=2.12.1" github.com/vektra/mockery/v2@v2.15.0
```

Comment:
I’ve never fully understood the `SemVer=2.12.1` bit of this line, but does it need updating too to match the new target version? (Or maybe we can drop it now?)

Reply:
Hm yeah, I'm not sure either! But might be good to have the versions match??

Reply:
It's a thing that was added to mockery to verify its own version. It's not great but it's needed today.

Reply:
Thanks @evanphx , I updated to match `2.15.0` in d337936bf